### PR TITLE
Add Psychonauts 2 GOG version to database

### PR DESCRIPTION
**Pull request by Prodeguerriero**

### CODE REVIEW

File: gamedb.yaml [hunk 1]
```diff
@@ -5069,6 +5069,15 @@
   compat_tool: proton-stable
   launch_options: -force -dx11
 
+- name: "Psychonauts 2"
+  platform: gog
+  id: 1508046796
+  status: playable
+  banner: steam:607080
+  poster: https://cdn2.steamgriddb.com/thumb/7d5b05b45008485350016d586a6a3e2e.jpg
```
Comment (alkazar) [inline gamedb.yaml:5077]: Don't need poster, `banner: steam:607080` is a shorthand which grabs all the artwork from Steam.

File: gamedb.yaml [hunk 2]
```diff
@@ -5069,6 +5069,15 @@
   compat_tool: proton-stable
   launch_options: -force -dx11
 
+- name: "Psychonauts 2"
+  platform: gog
+  id: 1508046796
```
Comment (alkazar) [inline gamedb.yaml:5074]: Is this the id showing in the URL of the game in the Chimera web app?
https://gogdb.org says this is the package id, and the game id is `1731504234`

**Review by alkazar — CHANGES_REQUESTED** [review]

File: gamedb.yaml (hunk 1 above)
Comment (Prodeguerriero) [reply gamedb.yaml:5077]: Roger that. Is this documented anywhere? I can create a README file for this repo, but I am afraid I don't know enough about the inner workings of the repo to write something exhaustive.

File: gamedb.yaml (hunk 2 above)
Comment (Prodeguerriero) [reply gamedb.yaml:5074]: I'll check. I remember looking at the database and picking the ID for a reason. I don't remember what it was, though :D

Comment (Prodeguerriero) [reply gamedb.yaml:5074]: Sorry for the long wait. The Chimera web app does indeed show `1731504234`. I'll fix it.

File: gamedb.yaml (hunk 1 above)
Comment (alkazar) [reply gamedb.yaml]: Yeah, it is not documented. It is on my TODO list.

**Comment by alkazar** [comment]
Looks good now, thanks!

**Review by alkazar — APPROVED** [review]

**Pull request merged**